templates/layouts/layout.html.twig

```diff
@@ -131,12 +131,14 @@
                             </div>
                         </li>
                         <li class="nav-item dropdown{% if menuSlug == 'contribute' %} active{% endif %}">
-                            <a class="nav-link dropdown-toggle" href="{{ site.url }}/contribute/" id="navbarContributeDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Contribute</a>
+                            <a class="nav-link dropdown-toggle" href="{{ site.url }}/contribute/" id="navbarDevelopmentDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Development</a>
 
                             <div class="dropdown-menu" aria-labelledby="navbarContributeDropdown">
                                 <a class="dropdown-item" href="{{ site.url }}/contribute/">Contributor Workflow</a>
                                 <a class="dropdown-item" href="{{ site.url }}/contribute/maintainer/">Maintainer Workflow</a>
                                 <a class="dropdown-item" href="{{ site.url }}/contribute/website/">Contribute to Website</a>
+                                <a class="dropdown-item" href="{{ site.url }}/deprecation-policy/">Deprecation Policy</a>
```

Comment:
Should we have a link with all of our policies? Otherwise the menu might get a bit big as time passes by

Reply:
Sure, I can go ahead and setup that structure to support other policies like releases, security, etc.

Reply:
Awesome, thanks!

Reply:
Note: releases is something I'm currently working on :-)